src/components/providers/authentication-provider.tsx

```diff
@@ -84,8 +86,19 @@ const AuthenticationProvider = ({ children }: Props) => {
     setPersons(fetchedPersons);
   };
 
+  /**
+   * fetchs avatars
+   */
+  const getSlackAvatars = async () => {
+    const fetchedAvatars = await slackAvatarsApi.slackAvatar();
+    setAvatars(fetchedAvatars);
+  };
+
   useEffect(() => {
-    if (auth) getPersonsList();
+    if (auth) {
+      getPersonsList();
+      getSlackAvatars();
```

Comment:
Should this getSlackAvatars really be in the authentication provider, its not related imo. Perhaps this should be in the nav bar?